libs/tup-components/src/tickets/TicketsTable.tsx

```diff
@@ -80,24 +75,32 @@ export const TicketsTable: React.FC<{ ticketsPath: string }> = ({
           </tr>
         </thead>
         <tbody>
-          {ticketData.map((ticket) => (
-            <tr
-              key={ticket.numerical_id}
-              className={
-                ticket.Status === 'user_wait' ? 'ticket-reply-required' : ''
-              }
-            >
-              <td>{ticket.numerical_id}</td>
-              <td>
-                <Link to={`${ticketsPath}/${ticket.numerical_id}`}>
-                  {ticket.Subject || '(No Subject)'}
-                </Link>{' '}
-                <AttachmentIndicator ticketId={ticket.numerical_id} />
+          {ticketData && ticketData.length ? (
+            ticketData.map((ticket) => (
+              <tr
+                key={ticket.numerical_id}
+                className={
+                  ticket.Status === 'user_wait' ? 'ticket-reply-required' : ''
+                }
+              >
+                <td>{ticket.numerical_id}</td>
+                <td>
+                  <Link to={`${ticketsPath}/${ticket.numerical_id}`}>
+                    {ticket.Subject || '(No Subject)'}
+                  </Link>{' '}
+                  <AttachmentIndicator ticketId={ticket.numerical_id} />
+                </td>
+                <td>{formatDate(new Date(ticket.Created))}</td>
+                <td>{getStatusText(ticket.Status)}</td>
+              </tr>
+            ))
+          ) : (
```

Comment:
Instead of these big ternary statements, let's use an early return if the list is empty. You can see examples in how we handle loading/error states: 
<img width="564" alt="image" src="https://user-images.githubusercontent.com/12601812/226650716-d2df83fa-b69c-4739-98d5-30ee7fd548f9.png">